test/unit/network_spec.js

```diff
@@ -69,11 +69,10 @@ describe('network', function() {
     // The test is valid for FF only: the XHR has support of the
     // 'moz-chunked-array' response type.
     // TODO enable for other browsers, e.g. when fetch/streams API is supported.
-    var m = /Mozilla\/5.0.*?rv:(\d+).*? Gecko/.exec(userAgent);
-    if (!m || m[1] < 9) {
-      expect(true).toEqual(true);
-      done();
-      return;
+    var isFirefoxWithMozChunkedEncodingSupport = /Mozilla\/5.0.*?rv:(\d+).*? Gecko/.exec(userAgent);
+    var isFetchWithStreamSupport = typeof ReadableStream;
```

Comment:
Again, this variable doesn't actually contain a boolean. Furthermore, this condition doesn't look right (compare with https://github.com/mozilla/pdf.js/commit/cd95b426c730108a529fbf6ab48b3192e22922dd). Hence you'd want something like this:

```javascript
let isFetchWithStreamSupport = (typeof Response !== 'undefined' &&
                                'body' in Response.prototype &&
                                typeof ReadableStream !== 'undefined');